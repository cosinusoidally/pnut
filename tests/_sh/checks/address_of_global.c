// expect_comp_failure
int a;

void main() {
  int* b = &a;
}
