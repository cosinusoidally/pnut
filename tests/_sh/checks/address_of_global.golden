tests/_sh/checks/address_of_global.c:7:0  comp_rvalue_go: can't take the address of a local variable
